mlflow/sagemaker/cli.py

```diff
@@ -49,8 +49,9 @@ def run_local(model_path, run_id, port, container):
 @click.option("--build/--no-build", default=True, help="Build the container if set.")
 @click.option("--push/--no-push", default=True, help="Push the container to amazon ecr if set.")
 @click.option("--container", "-c", default=IMAGE, help="image name")
+@click.option("--version", "-v", default="latest", help="version to tag the image with")
```

Comment:
What is the motivation for having version as a parameter? Shouldn't it always point to the same version of MLflow?

Reply:
the version of MLflow in mlflow.version.VERSION is the most recent *released* version, right? i think most likely we will be trying to create images for the *next* version when using this utility.

Reply:
(this relates one of the key questions i have in #126 as well)